podpac/core/node.py

```diff
@@ -103,13 +102,27 @@ class Node(tl.HasTraits):
      * ``_output``: the output of the most recent call to eval
     """
 
+    outputs = tl.List(tl.Unicode, allow_none=True).tag(attr=True)
+    output = tl.Unicode(default_value=None, allow_none=True).tag(attr=True)
     units = tl.Unicode(default_value=None, allow_none=True).tag(attr=True)
     dtype = tl.Any(default_value=float)
     cache_output = tl.Bool()
     cache_update = tl.Bool(False)
     cache_ctrl = tl.Instance(CacheCtrl, allow_none=True)
     style = tl.Instance(Style)
 
+    @tl.default("outputs")
+    def _default_outputs(self):
+        return None
+
+    @tl.validate("output")
+    def _validate_output(self, d):
+        if self.outputs is None:
```

Comment:
Strange that you don't need `and d['value'] is not None`.... but fine.

Reply:
I think that the validator isn't called when the default value is supplied, or something like that, but if someone supplied `=None` intentionally, you are probably right.